cypress/integration/ete/registration/register.2.cy.ts

```diff
@@ -9,7 +9,7 @@ describe('Registration flow', () => {
 			cy.intercept('GET', googleTermsUrl, (req) => {
 				req.reply(200);
 			});
-			cy.visit('/signin?useIdapi=true');
+			cy.visit('/register/email?useIdapi=true');
```

Comment:
because it should've been `/register` all along, as these are registration tests!

it seems someone copied them from the sign in tests, but didn't replace the url